big_tests/tests/muc_SUITE.erl

```diff
@@ -2667,7 +2668,21 @@ change_availability_status(ConfigIn) ->
     end).
 
 
-%Missing Direct Invitations (no examples)
+% Direct Invitations (example 1 from XEP-0249)
+
+direct_invite(ConfigIn) ->
+    UserSpecs = [{alice, 1}, {bob, 1}, {kate, 1}],
+    story_with_room(ConfigIn, [], UserSpecs, fun(Config, Alice, Bob, Kate) ->
+        escalus:send(Alice, stanza_muc_enter_room(?config(room, Config), nick(Alice))),
+        escalus:wait_for_stanzas(Alice, 2),
+        escalus:send(Alice, stanza_direct_invitation(?config(room, Config), Alice, Bob)),
+        escalus:send(Alice, stanza_direct_invitation(?config(room, Config), Alice, Kate)),
+        %Bob ignores the invitation, Kate accepts
+        is_direct_invitation(escalus:wait_for_stanza(Bob)),
+        is_direct_invitation(escalus:wait_for_stanza(Kate)),
+        escalus:send(Kate, stanza_muc_enter_room(?config(room, Config), nick(Kate))),
+        escalus:wait_for_stanzas(Kate, 2)
```

Comment:
It would be good to check what's inside these stanzas, because these could be errors as well.